ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -4,6 +4,30 @@
 from ivy.functional.frontends.paddle.func_wrapper import to_ivy_arrays_and_back
 
 
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def batch_norm(
+    x,
+    weight=None,
+    bias=None,
+    running_mean=None,
+    running_var=None,
+    training=False,
+    momentum=0.1,
+    epsilon=1e-05,
+    name=None,
```

Comment:
```suggestion
    x,
    running_mean,
    running_var,
    weight,
    bias,
    training=False,
    momentum=0.9,
    epsilon=1e-05,
    data_format="NCHW",
    use_global_stats=None,
    name=None,
```
This should be the function signature, following the paddle docs here https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/batch_norm_en.html#batch-norm